Eevee.Sleep.Bot/Extensions/DiscordExtensions.cs

```diff
@@ -27,6 +28,26 @@ public static class DiscordExtensions {
             .SendMessageAsync(message, embed: embed, embeds: embeds);
     }
 
+    private static Task<IMessageChannel> GetInGameAnnouncementNoticeChannelsAsync(this IDiscordClient client, InGameAnnoucementLanguages language) {
+        return language switch {
```

Comment:
See the note in `ConfigHelper`. After the change, calls involving `switch` shouldn't be needed here.